inspirehep/modules/records/cli.py

```diff
@@ -374,6 +378,26 @@ def _gen_query(query, page_start=1, page_end=-1, window_size=100):
             query = None
 
 
+class MyThreadPool(ThreadPool):
```

Comment:
I set this name to be sure that no one uses it anywhere else. As this is a dummy ThreadPool only for this CLI task.